CodeEdit/Features/WindowCommands/FileCommands.swift

```diff
@@ -44,12 +44,16 @@ struct FileCommands: Commands {
             .keyboardShortcut("w", modifiers: [.control, .shift, .command])
 
             Button("Close Window") {
-
+                NSApp.keyWindow?.close()
             }
             .keyboardShortcut("w", modifiers: [.shift, .command])
 
             Button("Close Workspace") {
-
+                NSDocumentController().closeAllDocuments(
```

Comment:
This closes all documents, that's not the expected behavior

Reply:
closing all the documents in this case means closing the workspace